__tests__/unstated.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from 'react';
```

Comment:
What is this file for?

Reply:
@thejameskyle for the JSX stuff, this is the way you would do it using TS (instead of `import React from 'react')

Reply:
I'm asking about the file, not the import

Reply:
These are the "tests" for TypeScript :) We did this in `material-ui` too (https://github.com/mui-org/material-ui/tree/v1-beta/test/typescript). It is the easiest way to prevent regressions. For example, you could add the `typescript` task to a CI/CD to verfiy nothing broke.

Reply:
Can you add it to the travis build then?

Reply:
done